### DISCUSSION TITLE
Blocking information flows.

### DISCUSSION THREAD

**Discussion by physicsman** [OpenMined/courses#163] [General]
I think I have my project idea. 

<img width="594" alt="Screen Shot" src="https://user-images.githubusercontent.com/4141441/103827042-4ec5b280-5035-11eb-8eec-869d1e7bb2f1.png">

![ErPwfU0VoAAbj0E](https://user-images.githubusercontent.com/4141441/104075124-22459e00-51cf-11eb-82e4-1918e14b567d.jpg)

**Comment by yemikifouly** [comment]
Hahaha... That sounds like a great project idea Jason! 👍

**Comment by em-blue** [comment]
Great idea! An interesting question here is: is the goal to block it? Would that really help anything? Or is the issue that we're missing verification on a lot of the information being presented here? Or... is this a downstream effect of a different broken information flow: of poor verification of vote counting? lots to think about - it's an extremely important problem! You'd save the world if you could solve it @physicsman

**Reply by physicsman** [reply]
The goal is to allow people to listen to whomever they choose. A social media space with a de facto monopoly on access to important public figures and government deciding 'who is told what' is unacceptable.

**Reply by em-blue** [reply]
If you're interested in some research/work done to address this (huge) issue, check out Nate Matias' work on CivilServant https://civilservant.io/about_us.html

**Reply by physicsman** [reply]
I appreciate the link unfortunately they appear stuck in the same pseudo reality as everywhere else atm.
